website/versioned_docs/version-0.3.x/client-setup.mdx

```diff
@@ -65,14 +67,36 @@ More details: [`ComposeClient`](./api/classes/client.ComposeClient.md)
 
 The [`ComposeClient`](./api/classes/client.ComposeClient.md) instance created in the previous step can be used to execute GraphQL queries using a schema automatically generated from the runtime composite definition.
 
-```js
-await compose.executeQuery(`
-  query {
-    viewer {
-      id
+After adding the code below to your app, you can run it with the command `node ./src/compose.js` from your terminal.
+
+```js title="src/compose.js"
+async function main() {
+  const result = await compose.executeQuery(`
+    query {
+      simpleProfileIndex(first: 10) {
+        edges {
+          node {
+            displayName
+          }
+        }
+      }
     }
+  `)
+
+  for (let node of result.data.simpleProfileIndex.edges) {
+    const properties = JSON.parse(JSON.stringify(node))
```

Comment:
Why do you use JSON parse/stringify here please? The data should already be JS objects that can be logged, no?